src/pages/MainPage/MainPage.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../styles/variables';
```

Comment:
It's better to use CSS-modules, rather than just CSS. It not crucial for such small app, but when you create something much bigger...